userscripts/ExTagLink.user.js

```diff
@@ -74,10 +75,27 @@ for (const com of $$('.c6')) {
       pixivLinks.push(pixivLink);
     }
 
+    const fanboxLink = textNode.textContent.match(
+      /https:\/\/\S+\.fanbox\.cc\/\S+/g
+    );
+
+    if (fanboxLink) {
+      fanboxLinks.push(fanboxLink);
+    }
+
     textNode = walker.nextNode();
   }
 }
 
+if (fanboxLinks.length) {
+  const linkItems = fanboxLinks.map((link) => ({
+    name: new URL(link).hostname.split('.')[0],
```

Comment:
I found there are 2 patterns in comments:

- `https://www.fanbox.cc/@lambda`
- `https://lambda.fanbox.cc/`

I think we can show the name `lambda` for both patterns

Reply:
good call, I was not aware of the other URL pattern. I hope I covered it with the latest update!